<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>managerDlg</class>
 <widget class="QWidget" name="managerDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_select">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>选择文件</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gamepath">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_up">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="gamename">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_del">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>删除文件</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="game_version">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="gam_os">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>版本</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>适用系统</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>游戏名字</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
